pkg/midstream/write.go

```diff
@@ -58,6 +58,11 @@ func (m *Midstream) WriteMidstream(options WriteOptions) error {
 		return errors.Wrap(err, "failed to write patches")
 	}
 
+	if m.Kustomization.CommonAnnotations == nil {
+		m.Kustomization.CommonAnnotations = make(map[string]string)
+	}
+	m.Kustomization.CommonAnnotations["kots.io/app-slug"] = options.AppSlug
+
```

Comment:
maybe add a comment that merge kustomization is ignored on headless install?

Reply:
this is actually happens if initial config screen is bypassed.  so an app with no required configuration will also have this problem.